tools/startup/apt-deps

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+verb=$1
+
+if [[ "$verb" = "generate" ]]; then
+    infile=$2
+    outfile=$3
+
+    if [[ ! -f "$infile" ]] || [[ -z "$outfile" ]]; then
+      echo "Should supply input and output files: $0 $1 [input] [output]"
+      exit 1;
+    fi
+
+    echo "Running apt-get update, installing wget"
+    apt-get update &> /dev/null
+    apt-get install -y wget &> /dev/null
```

Comment:
Do you think we could easily make it so that if `wget` is cached on the disk it takes it from there?

Reply:
@oferb I don't think it's needed. On the majority of distributions `apt-get install -y wget` will be a `no-op`. I only included it because I tested the script on Docker container, which is stripped from almost _everything_  🙁